crates/rolldown_common/src/inner_bundler_options/types/experimental_options.rs

```diff
@@ -11,10 +11,15 @@ use serde::Deserialize;
 )]
 pub struct ExperimentalOptions {
   pub strict_execution_order: Option<bool>,
+  pub disable_live_bindings: Option<bool>,
 }
 
 impl ExperimentalOptions {
   pub fn is_strict_execution_order_enabled(&self) -> bool {
     self.strict_execution_order.unwrap_or(false)
   }
+
+  pub fn is_disable_live_bindings_enabled(&self) -> bool {
```

Comment:
the naming is a little verbose, I think `disable_live_bindings` is enough.

Reply:
Just follow the previous `is_strict_execution_order_enabled` pattern. But yeah, shorthand pattern is also fine.

Reply:
I am ok for both, just a NIT